samples/Common.c

```diff
@@ -914,6 +913,8 @@ STATUS createSampleConfiguration(PCHAR channelName, SIGNALING_CHANNEL_ROLE_TYPE
     pSampleConfiguration->iceCandidatePairStatsTimerId = MAX_UINT32;
     pSampleConfiguration->pregenerateCertTimerId = MAX_UINT32;
     pSampleConfiguration->signalingClientMetrics.version = SIGNALING_CLIENT_METRICS_CURRENT_VERSION;
+    pSampleConfiguration->audioCodec = audioCodec;
```

Comment:
Can we set these up in the samples directly instead of here after the `createSampleConfiguration` call? Only because we are changing the function signature.

Reply:
Done!